src/com/xilinx/rapidwright/design/DesignTools.java

```diff
@@ -2225,4 +2231,50 @@ public static void makePhysNetNamesConsistent(Design design) {
         
         return path;
     }
+
+	/**
+	 * Create a Job running Vivado to create a readable version of
+	 * the EDIF inside the checkpoint to a separate file.
+	 * @param checkpoint the input checkpoint
+	 * @param edif the output EDIF
+	 * @return the created Job
+	 */
+	public static Job generateReadableEDIFJob(Path checkpoint, Path edif) {
+		try {
+
+			final Job job = new LocalJob();
+			job.setCommand(FileTools.getVivadoPath() + " -mode batch -source readable.tcl");
+
+			final Path runDir = Files.createTempDirectory(edif.toAbsolutePath().getParent(),edif.getFileName()+"_readable_edif_");
```

Comment:
It appears that the temporary directory of files does not get deleted after things finish.  I can see a case to keeping it if EDIF conversion fails, however, it should probably be deleted if the job completes successfully.